README.md

```diff
@@ -20,7 +20,7 @@ Hope to see you there!
 | Date (MM/DD/YYYY) | Presenter | Title | Links |
 | ----------------- | --------- | ----- | ----- |
 | 3/8/2023 | Nate | Hypothesis Testing for Error Mitigation: How to Evaluate Error Mitigation | [paper](https://arxiv.org/abs/2301.02690) |
-| 3/1/2023 | Purva Thakre | Circuit knitting with classical communication | [paper](https://arxiv.org/abs/2205.00016) |
+| 3/1/2023 | Purva Thakre | Circuit knitting with classical communication | [paper](https://arxiv.org/abs/2205.00016) |, [slides](https://drive.google.com/file/d/1QWnmPV2BD9zh57AK829gf7g-E03jEA0y/view?usp=sharing)|
```

Comment:
Because of how the table is made, the slides need to go in the "Links" column. The suggestion below should work.

```suggestion
| 3/1/2023 | Purva Thakre | Circuit knitting with classical communication | [paper](https://arxiv.org/abs/2205.00016), [slides](https://drive.google.com/file/d/1QWnmPV2BD9zh57AK829gf7g-E03jEA0y/) |
```